Automattic/jetpack
# Show cart icon on Simple wp-admin

Fixes:
- https://github.com/Automattic/dotcom-forge/issues/7247

## Proposed changes:

* Added the cart icon to the masterbar
* The cart icon links to the checkout page
* The cart icon has the same behaviour as the Calypso counterpart.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply the changes on your sandbox and check a simple site.
* Using wp-admin... add a product to the cart and close the checkout page while leaving the items in the cart
* You should see the cart icon on wp-admin
* The cart icon should redirect to the checkout page.

![Screenshot 2024-09-10 at 17 11 09](https://github.com/user-attachments/assets/cc32bf7a-1374-4834-b666-e482d0922640)
